test/unit/utility/type_list/type_list_test.cpp

```diff
@@ -23,6 +23,6 @@ TEST(type_list, concept)
 {
     using t = seqan3::type_list<int, char, double>;
 
-    EXPECT_TRUE(seqan3::detail::type_list_specialisation<t>);
-    EXPECT_FALSE(seqan3::detail::type_list_specialisation<int>);
+    EXPECT_TRUE((seqan3::detail::template_specialisation_of<t, seqan3::type_list>));
+    EXPECT_FALSE((seqan3::detail::template_specialisation_of<int, seqan3::type_list>));
```

Comment:
Do we need these double brackets?

Reply:
I don't know exactly why, but sometimes you need these double brackets. In this case, I would get an error message, that there is only one argument instead of two.

Reply:
`EXPECT_FALSE` is a macro and the preprocessor would interpret a comma differently, unless it's enclosed in parentheses